src/main/webapp/app/exercises/quiz/manage/apollon-diagrams/apollon-diagram-create-form.component.html

```diff
@@ -28,7 +28,7 @@ <h4 class="modal-title" id="myApollonDiagramLabel" jhiTranslate="artemisApp.apol
             <fa-icon [icon]="faBan"></fa-icon>&nbsp;
             <span jhiTranslate="entity.action.cancel">Cancel</span>
         </button>
-        <button type="submit" [disabled]="editForm.form.invalid || isSaving" class="btn btn-primary">
+        <button id="save-dd-quiz" type="submit" [disabled]="editForm.form.invalid || isSaving" class="btn btn-primary">
```

Comment:
what does dd stand for?

Reply:
drag & drop. I updated it as "save-dnd-quiz"